src/hooks/useLogout.tsx

```diff
@@ -29,6 +29,11 @@ export const useLogout = (): LogoutHook => {
       setIsLoggingOut(true);
       await clearAuthInfo();
       setProducts([]);
+      setFeatures({
+        REQUIRE_OTP: true,
+        TRANSACTION_GROUPING: true,
+        DIST_ENV: ""
+      });
```

Comment:
Since we're always populating this from `/version` call, do we need to specify these values? Can we just have it be an empty object?

Reply:
okay, was thinking to reset them to the default values from BE, but i think an empty object would be better